process/blanket_library.py

```diff
@@ -31,14 +31,16 @@ def primary_coolant_properties(self, output: bool):
         References: see pumppower function description
         """
 
-        coolant = 0
-        if f2py_compatible_to_string(fwbs_variables.fwcoolant) == "helium":
-            coolant = 1
-        elif f2py_compatible_to_string(fwbs_variables.fwcoolant) == "water":
-            coolant = 2
-
-        if (fwbs_variables.coolwh == 0 and coolant == 1) or (
-            fwbs_variables.coolwh == 1 and coolant == 0
+        # Make sure that, if the inputs for the FW and blanket inputs are different,
+        # the ipump variable is appropriately set for seperate coolants
+        if (
+            f2py_compatible_to_string(fwbs_variables.fwcoolant) == "Helium"
```

Comment:
We are **sure** these strings will be titleized? Would it be safe to `.title` it to be safe?